lib/models/timer_type.dart

```diff
@@ -0,0 +1,4 @@
+enum TimerType {
```

Comment:
I propose to change this name because it does not correspond to its essence.

I suggest changing the name to: ApproachType